shop-inc/identity-management
# bg(authentication): add exception when user is not found

#### What does this PR do?
Fixes bug that occurs during login.

#### Description of Task to be completed?
 - When NODE_ENV=test, PROTOBUF_USER=protobuf-user by default if not found in environment variables.
 - Add UserNotFound custom exception
 - Check if the user is found during login.

#### How should this be manually tested?
With the corresponding protobuf file [identity.proto](https://github.com/shop-inc/proto-broker/blob/develop/identity.proto) make a call to the `loginUser` RPC with a valid Google ID token of a user who is not registered in our graph database.
The callback should be called with an error `User not found!`.

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
* [#164824251](https://www.pivotaltracker.com/story/show/164824251)
#### Screenshots (if appropriate):
N/A
